docs/source/tutorials/benchmarking.rst

```diff
@@ -44,6 +44,39 @@ This configuration computes the throughput and performance metrics on CPU and GP
     seed: 0
     image_size: 256
 
+You can also replace the keys from within the benchmarking script. To do this just use the value instead of passing an array. Taking the example of the folder dataset above, the configuration file can be modified as.
```

Comment:
~~You can also replace the keys from within the benchmarking script.~~ → `Additionally, the keys can be replaced from the benchmarking script.`